README.md

```diff
@@ -1,30 +1,103 @@
-## Synopsis
+# FreeOrion
 
-Source code repository of the FreeOrion project.
+FreeOrion is a free, Open Source, turn-based space empire and galactic conquest
+computer game.
 
-FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game. 
+FreeOrion is inspired by the tradition of the Master of Orion games, but does
+not try to be a clone or remake of that series or any other game.  It builds
+on the classic *4X* model by incorporating the nation-building elements of games
+such as *Europa Universalis 2*.
 
-## Vision Statement
+While its modular, Open Source design allows for a significant degree of
```

Comment:
Why not? Does the current vision statement describe the goal of the project?  Then it should be removed.  Don't you like the phrasing? Then it should be rewritten.